services/venice-router/src/test/java/com/linkedin/venice/router/stats/RouterMetricEntityTest.java

```diff
@@ -58,20 +58,35 @@ public void testRouterMetricEntities() {
                 VeniceMetricsDimensions.VENICE_STORE_NAME,
                 VeniceMetricsDimensions.VENICE_CLUSTER_NAME,
                 VeniceMetricsDimensions.VENICE_REQUEST_METHOD,
+                VeniceMetricsDimensions.HTTP_RESPONSE_STATUS_CODE,
                 VeniceMetricsDimensions.HTTP_RESPONSE_STATUS_CODE_CATEGORY,
                 VeniceMetricsDimensions.VENICE_RESPONSE_STATUS_CODE_CATEGORY)));
     expectedMetrics.put(
-        RouterMetricEntity.CALL_KEY_COUNT,
+        RouterMetricEntity.INCOMING_KEY_COUNT,
```

Comment:
Can you explain this change actually?  What is the semantic that's trying to be conveyed in this name change?

Reply:
I see.  Let's document that, maybe even with a bit of javadoc. I'm not sure I would understand this when looking at a dashboard, and I'm not sure I'm clever enough to suggest a rename that would capture this nuance.

Reply:
`MetricEntity` has a description field which has this info. Made it more explicit and added java doc as well.

Reply:
Thought more about this and looking at further code, it looks like the request side and response side code for metric handling are not isolated and having separate metrics for incoming vs processed is only making it more confusing. Reverted some of the changes to remove incoming context and currently having only `call_count` and `key_count`.